libs/dapp-types/src/ChainId.ts

```diff
@@ -45,30 +42,91 @@ export enum EVMChainId {
 
 // Pre-calculated TypedChainId values that are supported in the dapp
 export enum PresetTypedChainId {
-  EthereumMainNet = calculateTypedChainId(ChainType.EVM, EVMChainId.EthereumMainNet),
-  Rinkeby = calculateTypedChainId(ChainType.EVM, EVMChainId.Rinkeby),
-  Ropsten = calculateTypedChainId(ChainType.EVM, EVMChainId.Ropsten),
-  Kovan = calculateTypedChainId(ChainType.EVM, EVMChainId.Kovan),
+  EthereumMainNet = calculateTypedChainId(
+    ChainType.EVM,
+    EVMChainId.EthereumMainNet
+  ),
+
   Goerli = calculateTypedChainId(ChainType.EVM, EVMChainId.Goerli),
+
   Sepolia = calculateTypedChainId(ChainType.EVM, EVMChainId.Sepolia),
-  HarmonyTestnet1 = calculateTypedChainId(ChainType.EVM, EVMChainId.HarmonyTestnet1),
-  HarmonyTestnet0 = calculateTypedChainId(ChainType.EVM, EVMChainId.HarmonyTestnet0),
-  HarmonyMainnet0 = calculateTypedChainId(ChainType.EVM, EVMChainId.HarmonyMainnet0),
+
+  HarmonyTestnet1 = calculateTypedChainId(
+    ChainType.EVM,
+    EVMChainId.HarmonyTestnet1
+  ),
+
+  HarmonyTestnet0 = calculateTypedChainId(
+    ChainType.EVM,
+    EVMChainId.HarmonyTestnet0
+  ),
```

Comment:
I think we only need one harmony testnet, no? What are the differences?

Reply:
Can you help me to confirm this, please? This config is existing before I edited this file. @AhmedKorim

Reply:
we should  keep both, 0,1 are related to network shard https://docs.harmony.one/home/general/ecosystem/wallets/browser-extensions-wallets/others/adding-harmony